frontend/views/containers/chatroom/poll-message-content/PollVoteResult.vue

```diff
@@ -0,0 +1,209 @@
+<template lang='pug'>
+.c-poll-voted
+  .c-content-header
+    i18n.c-poll-label(tag='label') poll
+    h3.is-title-4 {{ pollData.question }}
+
+    i18n.pill.is-neutral.c-poll-expired-badge(v-if='isPollExpired') Poll closed
+    menu-parent.c-poll-menu(v-else)
+      menu-trigger.is-icon.c-poll-menu-trigger
+        i.icon-ellipsis-v
+      menu-content.c-poll-menu-content
+        ul
+          menu-item(tag='button' icon='edit' @click='() => pollUtils.switchOnChangeMode()')
```

Comment:
Did you test to see if you can write this `@click` handler without the function wrapper?

```pug
menu-item(tag='button' icon='edit' @click='pollUtils.switchOnChangeMode')
```

Reply:
@taoeffect good catch here, it does work without additional function wrapper.